go.mod

```diff
@@ -103,5 +104,5 @@ replace (
 	k8s.io/mount-utils => github.com/kcp-dev/kubernetes/staging/src/k8s.io/mount-utils v0.0.0-20220524063253-5bb0eeecf2cf
 	k8s.io/pod-security-admission => github.com/kcp-dev/kubernetes/staging/src/k8s.io/pod-security-admission v0.0.0-20220524063253-5bb0eeecf2cf
 	k8s.io/sample-apiserver => github.com/kcp-dev/kubernetes/staging/src/k8s.io/sample-apiserver v0.0.0-20220524063253-5bb0eeecf2cf
-	sigs.k8s.io/controller-runtime => github.com/kcp-dev/controller-runtime v0.11.3-0.20220601041701-6ae831c93ef5
+	sigs.k8s.io/controller-runtime => github.com/MatousJobanek/controller-runtime v0.11.3-0.20220615142232-b356c1c75aeb
```

Comment:
Undo 😄

Reply:
removed as the changes were not needed anymore